index.html

```diff
@@ -9,261 +9,393 @@
+                         <li>JavaScript (ES5/ES6)</li>
+                         <li>Ruby</li>
+                         <li>HTML</li>
+                         <li>CSS</li>
+                     </ul>
+                    </div>
+                   <div class="langs-img">
+                     <img class="images" src="images/hash.png" alt="#">
+                   </div>
+                 </div>
+                 <div class="frameworks">
+                     <div class="content-list">
+                         <h3>Frameworks</h3>
+                     <ul>
+                         <li>Bootstrap</li>
+                         <li>Ruby on rails</li>
+                         <li>RSpec</li>
+                         <li>Capibara</li>
+                         <li>Selenium</li>
+                     </ul>
+         
+                     </div>
+                     <img class="images" src="images/Group_18.png" alt="#">
+         
+                 </div>
+                 <div class="skills">
+                     <div class="content-list">
+                         <h3>Skills</h3>
+                     <ul>
+                         <li>Codekit</li>
+                         <li>Codepen</li>
+                         <li>Github</li>
+                         <li>GitLAB</li>
+                         <li>Terminal</li>
+                     </ul>
+                     </div>
+                     <img class="images" src="images/ic_skills.svg" alt="#">
+                     
+                 </div>
+                 <div class="langs-img-dsk">
+                    <img class="images" src="images/image_dots_with_teal.svg" alt="#">
+
+                 </div>
+             </div>
+
+             <div class="about-me-dsk">
+                <h3>About</h3>
+                <P>Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi ut aliquip ea eiusmod</P>
+        
+                <button name="view-project" type="button">
+                    <span class="project-view-btn">Get my CV</span>
+                    
+                    <span><img class="cv-btn" src="images/ic_download.png" alt="union"></span>
+                </button>
             </div>
-            <div class="project-lg">
-              <div class="project-lg-1">
-                <h3>Project name goes here</h3> 
-                
-                <div class="list-languages">
-                    <ul>
-                        <li><span>HTML/CSS</span></li>
-                        <li><span>Ruby on Rails</span></li>
-                        <li><span>JavaScript</span></li>
-                    </ul>
+             
+         </section>
+         <footer>
+            <div class="footer-container">
+                <div class="header-img-left">
+                    <img src="images/image-geometry_6.png" alt="image">
                 </div>
```

Comment:
- [ ] Kindly this image is related to the desktop version and it appears in the mob version.

| Your design  | Figma design |
| ------------- | ------------- |
| ![image](https://user-images.githubusercontent.com/35535130/155573917-3c55a37a-9efe-4b8b-a2c3-afaece74bf61.png) | ![image](https://user-images.githubusercontent.com/35535130/155573977-8f5c179d-d920-46e9-baac-c9ae6b7c2bdf.png) |